package/skelcd-control-SLES.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Jul 26 14:05:24 UTC 2016 - cwh@suse.com
+
+- Add labels explaining the system roles (fate#317481 comment#4)
```

Comment:
actually comment 44, but I think we should reference the P1 bug instead: ([bsc#989711](https://bugzilla.suse.com/show_bug.cgi?id=989711))